lib/improver/generate_ancillaries/generate_ancillary.py

```diff
@@ -239,10 +239,11 @@ def sea_mask(landmask, orog_band):
 
         coords = standard_orography.coords()
         if key == 'land':  # regular topographical bands above land
-            old_threshold, threshold = thresholds
-            orog_band = np.ma.masked_inside(
-                standard_orography.data, old_threshold,
-                threshold).mask.astype(int)
+            lower_threshold, upper_threshold = thresholds
+            orog_band = np.ma.masked_where(
```

Comment:
I think that it would be worth adding a note about this behaviour to the docstring. Something like: `For each topography band, the lower threshold is inclusive to the band whilst the upper threshold is exclusive i.e. lower_threshold <= band < upper_threshold.`

Reply:
I have put some stuff into the docstring to reflect this. I have also edited the example arrays illustrated in the docstring to emphasize the point.